src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets

```diff
@@ -55,19 +55,24 @@
                                TargetMonikers="@(TestNugetTargetMoniker)">
       <Output TaskParameter="ResolvedCopyLocalItems" ItemName="TestCopyLocal" />
     </PrereleaseResolveNuGetPackageAssets>
-
+    
     <ItemGroup Condition="'$(TestWithCore)' != 'true'">
-
       <_TestCopyLocalXunitFiles Include="$(PackagesDir)xunit.runner.console\2.1.0\tools\*.exe" />
       <_TestCopyLocalXunitFiles Include="$(PackagesDir)xunit.runner.console\2.1.0\tools\*.dll" />
-      <_TestCopyLocalXunitFiles Condition="'$(TestAppX)' == 'true'" Include="$(PackagesDir)microsoft.xunit.runner.uwp\**\*.*" />
-
       <TestCopyLocal Include="@(_TestCopyLocalXunitFiles)" >
         <NuGetPackageId>%(FileName)</NuGetPackageId>
         <NuGetPackageVersion>2.1.0</NuGetPackageVersion>
         <Private>false</Private>
       </TestCopyLocal>
     </ItemGroup>
+    
+    <ItemGroup Condition="'$(TestAppX)' == 'true'">
+      <_TestCopyLocalAppXRunner Include="$(PackagesDir)microsoft.xunit.runner.uwp\$(AppXRunnerVersion)\**\*.*"/>
```

Comment:
Where is AppxRunnerVersion set? We might need some validation or something somewhere for that if we expect it to be set in individual repos.